scripts/run-builder.sh

```diff
@@ -12,7 +12,13 @@ mkdir -pv ~/.docker
 cp -v /var/lib/docker/certs/client/* ~/.docker
 touch ./builder-started.txt
 bash ./scripts/setup_helm.sh
-bash ./scripts/setup_aws.sh $EKS_AWS_ACCESS_KEY_ID $EKS_AWS_ACCESS_KEY_SECRET $AWS_REGION $CLUSTER_NAME
+if [[ "$CLOUD_PROVIDER" == "aws" ]]; then
+  bash ./scripts/setup_aws.sh $EKS_AWS_ACCESS_KEY_ID $EKS_AWS_ACCESS_KEY_SECRET $AWS_REGION $CLUSTER_NAME
+elif [[ "$CLOUD_PROVIDER" == "do" ]]; then
+  bash ./scripts/setup_do.sh $DO_API_TOKEN $CLUSTER_NAME
+else
```

Comment:
I just think, the default case can be aws. Like if `CLOUD_PROVIDER` is missing then assume its running for aws. This will ensure all our existing deployments are not broken / does not needs any change.

Reply:
I have updated this as suggested.